pkg/manifests/manifests.go

```diff
@@ -2318,6 +2311,68 @@ func (f *Factory) ClusterMonitoringOperatorServiceMonitor() (*monv1.ServiceMonit
 	return sm, nil
 }
 
+func (f *Factory) MixinsEtcdPrometheusRule() (*monv1.PrometheusRule, error) {
+	r, err := f.NewPrometheusRule(f.assets.MustNewAssetReader(MixinsEtcdPrometheusRule))
+	if err != nil {
+		return nil, err
+	}
+
+	r.Namespace = f.namespace
+
+	if !f.config.ClusterMonitoringConfiguration.EtcdConfig.IsEnabled() {
```

Comment:
shouldn't the test be moved at the caller site?

Reply:
dunno, I just moved it to this place as it was previously in `PrometheusK8sRules()` - https://github.com/openshift/cluster-monitoring-operator/blob/master/pkg/manifests/manifests.go#L849-L857

Reply:
I agree. It was moved into `pkg/tasks/controlplane.go`